test/cypress/integration/payment-links/details-provided-in-query-params.cy.js

```diff
@@ -55,6 +55,29 @@ describe('Payment link visited with amount and reference provided as query param
       })
     })
   })
+
+  it('should show error page when confirmation page has a £0 amount and the "Continue to payment" button is clicked', () => {
```

Comment:
Need to check this with John's story.
I don't think you can get to this page when there is a 0 amount.
Throws an exception and the user will see an error page.

Reply:
As discussed over slack, will wait for UCD feedback etc